sormas-e2e-tests/src/test/resources/JsonFiles/InCountry-NoHospitalization-PostCase.json

```diff
@@ -0,0 +1,60 @@
+{
+    "caseClassification": "NOT_CLASSIFIED",
+    "uuid": "d955b4c1-2efe-40e3-ab1e-9780fcfa003f",
```

Comment:
why are we hardcoding UUIDs in json?

Reply:
The same way we are hardcoding in Service classes when using the object builder. 
It can be taken dynamically using other API endpoints but then we sould introduce a dependency into the test and the principle of isolation is not complied to.